From 09cbe67d3a5fea8774d529f41577e32b4b1ba81c Mon Sep 17 00:00:00 2001
Message-Id: <09cbe67d3a5fea8774d529f41577e32b4b1ba81c.1388073353.git.stefano.lattarini@gmail.com>
From: Stefano Lattarini <stefano.lattarini@gmail.com>
Date: Thu, 26 Dec 2013 16:52:06 +0100
Subject: [PATCH] tests: fix a spurious failure on Mac OS X

This change fixes bug#14706.

* lib/depcomp2.sh: Also cater to spurious diagnostic from GNU rm,
not only from Apple's rm.

Signed-off-by: Stefano Lattarini <stefano.lattarini@gmail.com>
---
 t/depcomp2.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/depcomp2.sh b/t/depcomp2.sh
index 3eba12d..3baaebf 100644
--- a/t/depcomp2.sh
+++ b/t/depcomp2.sh
@@ -47,7 +47,7 @@ $AUTOCONF
 cat stderr >&2
 # Ignore warning messages sometimes seen on Mac OS X; they are
 # not automake's fault anyway, but either autoconf's or Mac's.
-sed '/rm:.* conftest\.dSYM/d' stderr >stderr2
+sed '/rm:.*conftest\.dSYM/d' stderr >stderr2
 test -s stderr2 && { cat stderr2; exit 1; }
 
 :
-- 1.8.5.rc0.335.g7794a68
--- a/t/fort4.sh	2015-03-18 21:11:28.000000000 -0500
+++ b/t/fort4.sh	2015-03-18 21:11:44.000000000 -0500
@@ -62,6 +62,8 @@
 goodbye_SOURCES = bye.f90 sub/baz.f
 goodbye_FCFLAGS =
 LDADD = $(FCLIBS)
+distclean-local:
+	-rm -rf *.dSYM
 END

 $ACLOCAL
